modules/benchmarks/pom.xml

```diff
@@ -35,7 +35,6 @@
     <url>http://ignite.apache.org</url>
 
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <jmh.version>1.13</jmh.version>
         <javac.target>1.7</javac.target>
```

Comment:
is this correct?

Reply:
This is leftover properties section -- `javac.target` is no longer used, so it is removed, and jmh.version moved to parent.